exonum/src/runtime/rust/mod.rs

```diff
@@ -12,7 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Built-in Rust runtime module.
+//! The basic runtime with the native Rust services.
+//!
+//! This runtime is usually presents in every blockchain instance and can only process a static set
```

Comment:
>usually presents in every blockchain instance

Usually or every? And it's not clear what is "blockchain instance" (`Blockchain` object, the node, the network).